installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -3592,4 +3584,9 @@ UPDATE cf_PolicyTable SET policyDisplayName='CrowdStrike Found Low Vulnerabiliti
 
 UPDATE cf_PolicyTable SET severity='medium' WHERE policyId in ('GCP_Virtual_Machine_Disk_Encryption_CMK_Rule','GCP_Virtual_Machine_Disk_Encryption_CSEK_Rule','GCP_DataProc_CMK_Encryption','GCP_bigquery_table_encryption_cmks_rule','Cloud_Storage_should_be_encrypted_with_CMK','Use_CMK_to_encrypt_OS_and_Data_disk','Aws_appflow_encryption_using_KMS_CMKs_version-1_aws_KMS_CMKs_Encryption_appflow','Encrypt_GKE_Cluster_Node_with_CMK','GCP_PubSub_CMK_Encryption','Enable_Virtual_Machine_Disk_Volume_Customer_Managed_Key','Aws_dms_encryption_using_KMS_CMKs_version-1_aws_KMS_CMKs_Encryption_dms','Use_CMK_to_encrypt_Storage_Account_for_Activity_logs','AWS_ElasticSearch_Domain_Encryption_Using_CMK_KMS_Key_version-1_Enable_CloudTrail_Global_Services_cloudtrail');
 DELETE IGNORE FROM  cf_PolicyParams WHERE policyId='encrypt_os_and_data_disk';
-DELETE IGNORE FROM  cf_PolicyTable WHERE policyId='encrypt_os_and_data_disk';
\ No newline at end of file
+DELETE IGNORE FROM  cf_PolicyTable WHERE policyId='encrypt_os_and_data_disk';
```

Comment:
I believe these need to go to the migration file?

Reply:
@plyubich checked with Nithin, and the migration script is going in the next release. so, I think we can have these changes here going in this release